pkg/util/azureclient/storage/storage.go

```diff
@@ -0,0 +1,87 @@
+package storage
+
+import (
+	"io"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+)
+
+const (
+	// DefaultBaseURL is the domain name used for storage requests in the
+	// public cloud when a default client is created.
+	DefaultBaseURL = storage.DefaultBaseURL
+
+	// DefaultAPIVersion is the Azure Storage API version string used when a
+	// basic client is created.
+	DefaultAPIVersion = storage.DefaultAPIVersion
+)
+
+// Client is a minimal interface for azure Client
+type Client interface {
```

Comment:
nit: typically we name this interface as `Interface` instead of `Client` in the k8s/openshift client libraries.